src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -92,7 +92,7 @@ class ArrayField extends React.Component {
           },
         );
         focusElement(`[name="${scrollElementName}"] ${focusElementSelector}`);
-      }, 100);
+      }, window.Forms?.scroll ? 0 : 100);
```

Comment:
there is likely a much better way to determine if we are running in a unit test env.